src/main/resources/db/migration/0250/V282__DocumentProducerTables.sql

```diff
@@ -197,7 +197,7 @@ CREATE TABLE docprod.document_saved_versions (
 
 CREATE TABLE docprod.variable_values (
     id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
-    document_id BIGINT NOT NULL REFERENCES docprod.documents ON DELETE CASCADE,
+    project_id BIGINT NOT NULL REFERENCES projects,
```

Comment:
Should we have ON DELETE CASCADE for if a project gets deleted? 

Which actually prompts me to think about, is there any thing to stop a user from deleting an accelerator project?

Reply:
Ah, good call, not sure why I left that off here.

Currently you can't delete a project if it has accelerator details (there's no ON DELETE on the details table), though honestly I don't remember if that was intentional or an oversight.